installing/understanding-installations.adoc

```diff
@@ -0,0 +1,157 @@
+[id="understanding-installations"]
+= Understanding installations
```

Comment:
Not sure I would expect to find platform support information in "Understanding installations." Might want to run a quick user test to confirm this title + moving "Supported installation methods for different platforms" under it.

Reply:
@maxwelldb thanks for your comment, this is a good point. I will have a think about this.